server/webserver/__init__.py

```diff
@@ -0,0 +1,36 @@
+from flask import Flask
+import os
+
+
+def create_app(test_config=None):
+    app = Flask(__name__)
+    app.config["JWT_SECRET_KEY"] = os.urandom(24)
+    app.config["SQLALCHEMY_DATABASE_URI"] = "sqlite://"
+    app.config["SQLALCHEMY_TRACK_MODIFICATIONS"] = False
+    app.config["JWT_COOKIE_SECURE"] = True
+
+    from .extensions import jwt, db
+
+    jwt.init_app(app)
+    db.init_app(app)
+
+    from .blueprints.login import login
+
+    app.register_blueprint(login)
+
+    # @app.route("/")
+    # def homepage():
+    #     return """
+    #             <h1>This is the Homepage</h1>
+    #             <button><a href="/login">Login Here</a></button>
+    #            """
```

Comment:
oh yeah we're gonna set this up as the webserver too right? we'll need to sendFile() or render(), or sendHtml() or w/e it is to serve the static assets for index as a wildcard later

Reply:
oh yah, this bit of code was just to test out the Google Auth login button but I see what you mean, I think we can specify the static folder when instantiating the app e.g. `app = Flask(__name__, static_folder='/dist/')`

Reply:
Nice, however if we do that, we either gotta build it and commit it, or make sure CI can build and copy the artifacts over. Or oh yeah docker too